semgrep-core/matching/Semgrep_generic.ml

```diff
@@ -222,15 +222,22 @@ let apply_equivalences equivs any =
 (*****************************************************************************)
 
 (*s: function [[Semgrep_generic.check2]] *)
-let check2 ~hook rules equivs file lang ast =
+let check2 ~hook rules equivs file _lang ast =
```

Comment:
This was only here to support `Normalize_AST.normalize`, so we could undo the argument. 🤷

Reply:
Yep, except I've found a bug elsewhere that will require again this argument :)